<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog{
background-color: rgb(232, 232, 232);
}</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 0);
font: 81 16pt &quot;나눔스퀘어OTF ExtraBold&quot;;</string>
   </property>
   <property name="text">
    <string>뒤</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>680</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 24pt &quot;나눔스퀘어OTF ExtraBold&quot;;
border: 2px solid rgb(255, 170, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>기록</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>370</width>
     <height>360</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#qt_calendar_calendarview{
font: 81 12pt &quot;나눔스퀘어OTF ExtraBold&quot;;
border: 2px solid rgb(255, 170, 0);
color: rgb(0, 0, 0);
background-color: rgb(239, 239, 239);
selection-color: rgb(255, 255, 255);
selection-background-color: rgba(255, 170, 0, 100);
}

#qt_calendar_navigationbar{
background-color: rgba(255, 170, 0, 255);
min-height: 45px;
border: 2px solid rgb(255, 170, 0);
border-bottom: 0px;
}

#qt_calendar_yearbutton, #qt_calendar_monthbutton {
    color: rgb(0, 0, 0);
    min-width: 80px;
	font: 81 12pt &quot;나눔스퀘어OTF ExtraBold&quot;;
}
#qt_calendar_yearedit {
    color: rgb(0, 0, 0);	
	font: 81 12pt &quot;나눔스퀘어OTF ExtraBold&quot;;
	
	background-color: rgb(225, 150, 0);
}</string>
   </property>
   <property name="gridVisible">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QCalendarWidget::SingleSelection</enum>
   </property>
   <property name="horizontalHeaderFormat">
    <enum>QCalendarWidget::ShortDayNames</enum>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
   <property name="navigationBarVisible">
    <bool>true</bool>
   </property>
   <property name="dateEditEnabled">
    <bool>true</bool>
   </property>
   <property name="dateEditAcceptDelay">
    <number>1500</number>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stack">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>370</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="breakfast">
    <property name="styleSheet">
     <string notr="true">#breakfast{
border: 2px solid rgb(255, 170, 0);
}</string>
    </property>
    <widget class="QPushButton" name="pushButton1_1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>100</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어OTF ExtraBold</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::PreventContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>아침</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton1_2">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>20</y>
       <width>100</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어OTF ExtraBold</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>점심</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton1_3">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>20</y>
       <width>100</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어OTF ExtraBold</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>저녁</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="image">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>100</y>
       <width>205</width>
       <height>205</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 81 24pt &quot;나눔스퀘어OTF ExtraBold&quot;;
border: 2px solid rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>사진</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="kname">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>115</y>
       <width>110</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 81 10pt &quot;나눔스퀘어OTF ExtraBold&quot;;
border:1px solid rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>칼로리</string>
     </property>
    </widget>
    <widget class="QLabel" name="gname">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>215</y>
       <width>110</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 81 10pt &quot;나눔스퀘어OTF ExtraBold&quot;;
border:1px solid rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>무게</string>
     </property>
    </widget>
    <widget class="QLabel" name="kcal">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>150</y>
       <width>110</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(255, 255, 255);
font: 81 10pt &quot;나눔스퀘어OTF ExtraBold&quot;;</string>
     </property>
     <property name="text">
      <string>500(kcal)</string>
     </property>
    </widget>
    <widget class="QLabel" name="gram">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>250</y>
       <width>110</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(255, 255, 255);
font: 81 10pt &quot;나눔스퀘어OTF ExtraBold&quot;;</string>
     </property>
     <property name="text">
      <string>100(g)</string>
     </property>
    </widget>
    <widget class="QLabel" name="base">
     <property name="geometry">
      <rect>
       <x>233</x>
       <y>100</y>
       <width>125</width>
       <height>205</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>base</zorder>
    <zorder>pushButton1_1</zorder>
    <zorder>pushButton1_2</zorder>
    <zorder>pushButton1_3</zorder>
    <zorder>image</zorder>
    <zorder>kname</zorder>
    <zorder>gname</zorder>
    <zorder>kcal</zorder>
    <zorder>gram</zorder>
   </widget>
   <widget class="QWidget" name="lunch">
    <property name="styleSheet">
     <string notr="true">#lunch{
border: 2px solid rgb(255, 170, 0);
}</string>
    </property>
    <widget class="QPushButton" name="pushButton2_2">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>20</y>
       <width>100</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어OTF ExtraBold</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>점심</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton2_3">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>20</y>
       <width>100</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어OTF ExtraBold</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>저녁</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton2_1">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>100</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어OTF ExtraBold</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>아침</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="image_2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>100</y>
       <width>205</width>
       <height>205</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 81 24pt &quot;나눔스퀘어OTF ExtraBold&quot;;
border: 2px solid rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>사진</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="base_2">
     <property name="geometry">
      <rect>
       <x>233</x>
       <y>100</y>
       <width>125</width>
       <height>205</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border: 2px solid rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="kcal_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>150</y>
       <width>110</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(255, 255, 255);
font: 81 10pt &quot;나눔스퀘어OTF ExtraBold&quot;;</string>
     </property>
     <property name="text">
      <string>150(kacal)</string>
     </property>
    </widget>
    <widget class="QLabel" name="gname_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>215</y>
       <width>110</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 81 10pt &quot;나눔스퀘어OTF ExtraBold&quot;;
border:1px solid rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>무게</string>
     </property>
    </widget>
    <widget class="QLabel" name="kname_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>115</y>
       <width>110</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 81 10pt &quot;나눔스퀘어OTF ExtraBold&quot;;
border:1px solid rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>칼로리</string>
     </property>
    </widget>
    <widget class="QLabel" name="gram_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>250</y>
       <width>110</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(255, 255, 255);
font: 81 10pt &quot;나눔스퀘어OTF ExtraBold&quot;;</string>
     </property>
     <property name="text">
      <string>30(g)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="dinner">
    <property name="styleSheet">
     <string notr="true">#dinner{
border: 2px solid rgb(255, 170, 0);
}</string>
    </property>
    <widget class="QPushButton" name="pushButton3_2">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>20</y>
       <width>100</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어OTF ExtraBold</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>점심</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton3_3">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>20</y>
       <width>100</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어OTF ExtraBold</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>저녁</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton3_1">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>100</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어OTF ExtraBold</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>아침</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="image_3">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>100</y>
       <width>205</width>
       <height>205</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 81 24pt &quot;나눔스퀘어OTF ExtraBold&quot;;
border: 2px solid rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>사진</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="kcal_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>150</y>
       <width>110</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(255, 255, 255);
font: 81 10pt &quot;나눔스퀘어OTF ExtraBold&quot;;</string>
     </property>
     <property name="text">
      <string>1000(kacal)</string>
     </property>
    </widget>
    <widget class="QLabel" name="gram_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>250</y>
       <width>110</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid rgb(255, 255, 255);
font: 81 10pt &quot;나눔스퀘어OTF ExtraBold&quot;;</string>
     </property>
     <property name="text">
      <string>3000(g)</string>
     </property>
    </widget>
    <widget class="QLabel" name="kname_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>115</y>
       <width>110</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 81 10pt &quot;나눔스퀘어OTF ExtraBold&quot;;
border:1px solid rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>칼로리</string>
     </property>
    </widget>
    <widget class="QLabel" name="base_3">
     <property name="geometry">
      <rect>
       <x>233</x>
       <y>100</y>
       <width>125</width>
       <height>205</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="gname_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>215</y>
       <width>110</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 81 10pt &quot;나눔스퀘어OTF ExtraBold&quot;;
border:1px solid rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>무게</string>
     </property>
    </widget>
    <zorder>base_3</zorder>
    <zorder>pushButton3_2</zorder>
    <zorder>pushButton3_3</zorder>
    <zorder>pushButton3_1</zorder>
    <zorder>image_3</zorder>
    <zorder>kcal_3</zorder>
    <zorder>gram_3</zorder>
    <zorder>kname_3</zorder>
    <zorder>gname_3</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
